components/GiftCardDetails.js

```diff
@@ -184,7 +184,7 @@ class GiftCardDetails extends React.Component {
               {isExpired && (
                 <React.Fragment>
                   <Box mx={1}>|</Box>
-                  <FormattedMessage id="GiftCard.Expired" defaultMessage="Expired" />
+                  <FormattedMessage id="order.expired" defaultMessage="Expired" />
```

Comment:
I would rather add this one to `DUPLICATED_IGNORED_MESSAGES`. The exact translation depends on whether the subject (order/gift card) is feminine or masculine in the target language

Reply:
Added.